README.md

```diff
@@ -370,6 +370,7 @@ Security flags:
 - :whale: `--cap-add=<CAP>`: Add Linux capabilities
 - :whale: `--cap-drop=<CAP>`: Drop Linux capabilities
 - :whale: `--privileged`: Give extended privileges to this container
+- :blue_square: :nerd_face: `--host-process`: Allow running Host Process containers on Windows. Inherits permissions from containerd process unless `--user` is specified then will start with user specified.  The user specified must be present on the host.  Requires Containerd 1.6+.
```

Comment:
What about calling this `--isolation=host`

Reply:
this has grown on me overnight. I do have a concern on naming...  Right now we call this `hostprocess` containers in kubernetes and `job` containers in hcsshim and we would be adding another slightly different way of referencing this concept.  I think it is a minor difference but thought to bring it up. thoughts?

Reply:
The 'job containers' name was supposed to be just for the devs on hcsshim as it was what we got accustomed to when referring to it internally, although it does kind of suck that the name leaks out in some error strings. That should maybe get addressed/cleaned up. If that was touched up would that alleviate some of the concern on the three way split @jsturtevant?

Reply:
Feel like the UX is a bit improved if all the Windows folks need to remember is --isolation and its three options vs. --isolation and then an option off to the side that also dramatically reduces the amount of isolation you have. Also might simplify the case where both are specified (--isolation and --host-process) as ideally we should error out if they are as they're conflicting options.

Reply:
It's a minor concern. Just wanted to call it out. Anytime we refer to the same thing multiple ways it adds complexity to understanding. 

That said thinking about hostprocess as an `--isolation` resonates with me. and keeps it simple from the cli UX perspective.  

This brings up one more thought, As of right now hostprocess doesn't not support hyper-v containers. Is it possible in the future it could? Does that even make sense?

Reply:
That's a good question.. I could imagine a world where we could get it to work, but I'm not sure what use it would bring off the top of my head. I don't know if its worth putting too much brain power into thinking of at the moment though

Reply:
Just want to make sure we don't back our selves into a naming thing.  *If* it was a thing, would naming for isolation be something like `hyperv-host` :thinking:

Reply:
If we're able to specify --isolation twice (e.g. `--isolation host --isolation hyper-v`) would that suffice? We'd just need to short circuit on if someone specified process and a second option as that doesn't make sense. Or your idea of a fourth hyperv-host option.